internal/auth/auth.go

```diff
@@ -344,3 +352,68 @@ func persistConfiguration() error {
 	_ = os.MkdirAll(filepath.Dir(path), os.ModePerm)
 	return os.WriteFile(path, data, 0o600)
 }
+
+func RefreshCloudSession() error {
+	if globalConfig == nil {
+		c := DefaultConfig()
+		globalConfig = &c
+	}
+
```

Comment:
I think we should check if `globalConfig.CloudRefreshToken` is empty in which case just return an error that `cannot refresh session: require new login session`